verl/trainer/ppo/ray_trainer.py

```diff
@@ -69,24 +69,42 @@ class ResourcePoolManager:
 
     resource_pool_spec: dict[str, list[int]]
     mapping: dict[Role, str]
+    max_colocate_count: int = 1
     resource_pool_dict: dict[str, RayResourcePool] = field(default_factory=dict)
 
     def create_resource_pool(self):
         """Create Ray resource pools for distributed training.
 
         Initializes resource pools based on the resource pool specification,
         with each pool managing GPU resources across multiple nodes.
-        For FSDP backend, uses max_colocate_count=1 to merge WorkerGroups.
-        For Megatron backend, uses max_colocate_count>1 for different models.
+
+        Args:
+            max_colocate_count: Number of processes that can share a GPU.
+                - 1 (default): Each process gets exclusive GPU access (recommended for FSDP)
+                - >1: Multiple processes share GPUs (for Megatron with colocated models)
+                Note: max_colocate_count > 1 requires Ray >= 2.39.0 (PR #48088)
         """
+        # Check Ray version if max_colocate_count > 1
+        if self.max_colocate_count > 1:
+            from packaging.version import parse as parse_version
+            if parse_version(ray.__version__) < parse_version("2.39.0"):
```

Comment:
This is not right, ray support assign GPU fraction for actor (@ray.remote(num_gpus=0.1)) since very early version.

Reply:
Oh, yes, you're right. We have multiple ray versions in our internal environment, and quit a lot users are still using ray-2.10 which `placement_group_bundle_index` bug has been fixed. So this version check will break those user.

Can you just give a warning instead raise an exception?